index.html

```diff
@@ -222,5 +222,37 @@ <h2>Skills</h2>
             </div>
         </div>
     </section>
+    <section class="contact-form" id="contact">
+        <div class="contact-text">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <div class="contact-area">
+            <form action="https://formspree.io/f/xvolllzo" method="post" id="frmContact">
+              <label for="name">
+                <input type="text" name="name" id="name" maxlength="30" class="contact-input" placeholder="Enter your names..."
+                required/>
+              </label>
+              <label for="email">
+                <input type="email" name="email" id="email" class="contact-input" placeholder="Enter your gmail..."
+                required/>
+              </label>
+              <label for="text">
+                <textarea name="text" id="text" maxlength="500" class="contact-input" placeholder="Enter text here" cols="23" rows="5" required></textarea>
+              </label>
+                <input id="btn" type="submit" class="submit-btn dark-font" value="Get in Touch" />
+            </form>
+          </div>
+    </section>
+    <section class="footer">
+        <ul class="f-unordered">
+            <li class="f-list"><a href="" aria-label="github"><img src="./img/Vector-github.png" alt="github" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="linkedin"><img src="./img/vector-linkedin.png" alt="linkedin" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="angelist"><img src="./img/Vector-angelist.png" alt="angelist" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="twitter"><img src="./img/Vector-twitter.png" alt="twitter" class="img-icon"/></a></li>
+            <li class="f-list"><a href="" aria-label="medium"><img src="./img/Vector.png" alt="medium" class="img-icon"/></a></li>
+        </ul>
+    </section>
```

Comment:
- [ ] [optional] please make sure that your icons in the center of the page in order to match the Figma design